packages/tre/src/runtime/index.ts

```diff
@@ -86,11 +99,12 @@ class NativeRuntime implements Runtime {
 
     // 3. Write config
     runtimeProcess.stdin.write(configBuffer);
+    runtimeProcess.stdin.end();
   }
 
-  async dispose() {
+  dispose() {
```

Comment:
I know the `async` annotation isn't needed because you're directly returning a promise. It does make it clearer at a glance though that the function is async, and so I wonder explicitly annotating the return type here could serve that same function from a readability perspective?